api/src/main/data/bt_language_names.xml

```diff
@@ -414,6 +458,61 @@ QUESTIONS?
 <symbol>MALAYALAM</symbol>
 </language>
 <language>
+<id>107</id>
+<name>Marathi</name>
+<doc>Marathi</doc>
+<iso639-3>mar</iso639-3>
+<iso639-1>mr</iso639-1>
+<script>Deva</script>
+<symbol>MARATHI</symbol>
+</language>
+<language>
+<id>103</id>
+<name>Mongolian</name>
+<doc>Mongolian</doc>
+<iso639-3>mon</iso639-3>
+<iso639-1>mn</iso639-1>
+<script>Mong</script>
+<symbol>MONGOLIAN</symbol>
+<aux_name0>Mongolian (macrolanguage)</aux_name0>
+</language>
+<language>
+<id>104</id>
+<name>Mongolian, Halh</name>
+<doc>Halh Mongolian</doc>
+<iso639-3>khk</iso639-3>
+<script>Mong</script>
+<symbol>HALH_MONGOLIAN</symbol>
+</language>
+<language>
+<id>105</id>
+<name>Mongolian, Peripheral</name>
+<doc>Peripheral Mongolian</doc>
+<iso639-3>mvf</iso639-3>
+<script>Mong</script>
+<symbol>PERIPHERAL_MONGOLIAN</symbol>
+</language>
+<language>
+<id>100</id>
+<name>Nepali</name>
+<doc>Nepali</doc>
+<iso639-3>nep</iso639-3>
+<iso639-1>ne</iso639-1>
+<script>Deva</script>
+<symbol>NEPALI</symbol>
+<aux_name0>Nepali (macrolanguage)</aux_name0>
+</language>
+<language>
+</language>
+<language>
+<id>101</id>
+<name>Nepali(i)</name>
```

Comment:
Is “(i)” a conventional abbreviation for “(individual language)”? It looks weird, like the language is called both “Nepali” and “Nepalii”.

Reply:
I removed the (i) languages to avoid confusion.